packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/RancherDetail/RancherDetail.component.tsx

```diff
@@ -245,6 +257,49 @@ const RancherDetail = ({
                   isDisabled={!isReadyStatus}
                 />
               </TileBlock>
+              <div className="flex flex-col mb-3">
+                <div className="flex justify-between">
+                  <OsdsText
+                    className="mb-2"
+                    size={ODS_TEXT_SIZE._200}
+                    level={ODS_TEXT_LEVEL.heading}
+                    color={ODS_THEME_COLOR_INTENT.primary}
+                    hue={ODS_THEME_COLOR_HUE._800}
+                  >
+                    {t('egress_title')}
+                  </OsdsText>
+                  <OsdsTooltip>
+                    <OsdsIcon
+                      name={ODS_ICON_NAME.HELP}
+                      size={ODS_ICON_SIZE.xs}
+                      color={ODS_THEME_COLOR_INTENT.primary}
+                    />
+                    <OsdsTooltipContent slot="tooltip-content">
+                      {t('egress_tooltip')}
+                    </OsdsTooltipContent>
+                  </OsdsTooltip>
+                </div>
+                <OsdsText
+                  className="mb-2"
+                  size={ODS_TEXT_SIZE._400}
+                  level={ODS_TEXT_LEVEL.body}
+                  color={ODS_THEME_COLOR_INTENT.default}
+                >
+                  {/* {currentState.networking.egressCIDRBlocks.map((ip) => ( */}
```

Comment:
Should be removed, the code will be updated when the API will be available

Reply:
done